pkgs/development/python-modules/sphinx-lv2-theme/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitLab,
+  sphinx,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "sphinx-lv2-theme";
+  version = "1.4.2";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "LV2";
+    repo = "sphinx_lv2_theme";
+    rev = "v${version}";
+    hash = "sha256-x3h2hOQIyzOmnh58WMHjuOARdML1sDiVAf7m0YghhJ4=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
```

Comment:
```suggestion
  fetchFromGitLab,
  setuptools,
  sphinx,
}:

python3Packages.buildPythonPackage rec {
  pname = "sphinx-lv2-theme";
  version = "1.4.2";
  pyproject = true;

  src = fetchFromGitLab {
    owner = "LV2";
    repo = "sphinx_lv2_theme";
    rev = "v${version}";
    hash = "sha256-x3h2hOQIyzOmnh58WMHjuOARdML1sDiVAf7m0YghhJ4=";
  };

  build-system = [ setuptools ];
```

Reply:
Oh, it can be imported directly, never mind.

Reply:
Yes, all packages imported in `pkgs/top-level/python-packages.nix` will be provided with elements of the `python3Packages` package set. So, you can add them directly as arguments.